pkg/proxy/ipvs/proxier.go

```diff
@@ -647,29 +641,31 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 		return nil, fmt.Errorf("failed to find loaded kernel modules: %v", err)
 	}
 
-	builtinModsFilePath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersionStr)
-	b, err := ioutil.ReadFile(builtinModsFilePath)
+	bmods, nfconntrack, err = utilipvs.BuiltinIPVSModules(false, kernelVersionStr)
```

Comment:
The `bool` parameter to `BuiltinIPVSModules` is confusing too... what does `false` mean here?

Maybe have two functions? Or maybe `BuiltinIPVSModules` could figure out static vs dynamic itself (and indicate which in the return value? Having bool return values is a little easier to understand on the caller side because you can at least tell from the name of the variable it gets assigned to what it means...)

(Maybe what you were talking about with rewriting it to use a set would help)